tests/openssl/enc/enc.c

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/edger8r/enclave.h>
+#include <openenclave/enclave.h>
+#include <openssl/engine.h>
+#include <sys/mount.h>
+#include "openssl_t.h"
+#include "tu_local.h" /* Header from openssl/test/testutil */
+
+extern char** __environ;
+
+extern int main(int argc, char* argv[]);
+
+int enc_test(int argc, char** argv, char** env)
+{
+    int ret = 1;
+    ENGINE* eng = NULL;
+    const BIO_METHOD* tap = NULL;
+
+    /* Directly use environ from host. */
+    __environ = env;
+
+    /* Initialize socket and host fs. */
+    if (OE_OK != oe_load_module_host_socket_interface())
+        goto done;
+
+    if (OE_OK != oe_load_module_host_resolver())
+        goto done;
+
+#ifndef CODE_COVERAGE // Avoid conflicts with libgcov.
+    if (OE_OK != oe_load_module_host_file_system())
```

Comment:
What happens when code coverage is turned on? Will these tests still pass and test necessary assertions?

Reply:
Currently, libgcov always initializes the host file system by invoking the same function.  Commenting out here is to avoiding the duplicated invocation (i.e., `oe_load_module_host_file_system` will not return `OE_OK` for the second invocation).

Reply:
Thanks for the clarification. Add your explanation to the comment since it is more clearer.

Reply:
Done.